### DISCUSSION TITLE
Option to remove attr fill from <path>

### DISCUSSION THREAD

**Discussion by ghost** [marcomontalbano/figma-export#141] [Q&A] [answered]
Hi,

I'm using `@figma-export/output-components-as-svgstore` to generate an svg sprite, but wanted to know if there's an option to remove the `fill` attribute on `<path>`s? as I'm looking to control the colour with CSS.

Thanks

**Comment by ghost** [comment, marked as answer]
Solution found. https://github.com/svg/svgo/blob/main/plugins/removeAttrs.js

**Reply by marcomontalbano** [reply]
Thanks a lot @duytan-hoang-eurostar for sharing this!
Could you also share your `svgstoreConfig`? This will help others.
